app/pb_kits/playbook/pb_contact/_contact.jsx

```diff
@@ -26,15 +27,24 @@ const Contact = ({
   className,
   dark=false,
   contactValue,
+  contactDetail,
 }: ContactProps) => {
 
+  const formatDetail = (contactDetail) => {
+    if (contactDetail !== undefined) {
+      return `\u00b7 ${contactDetail}`
+    } else {
+      return ""
+    }
+  }
+
   const css = classnames(kitClasses({contactType}), className)
 
-  const formatPhoneNumber = (phoneNumberString, contactType) => {
+  const formatContact = (contactString, contactType) => {
     if (contactType == "email") {
-      return phoneNumberString
+      return contactString
     } else {
-      let cleaned = ('' + phoneNumberString).replace(/\D/g, '')
+      let cleaned = ('' + contactString).replace(/\D/g, '')
```

Comment:
Since we aren't mutating `cleaned`, use `const`

```suggestion
      const cleaned = (contactString).replace(/\D/g, '')
```

also, can `contactString` possibly come through as another type? Is that why we are forcibly stringifying here? `' ' + contactString...`

Reply:
No it looks like that's going to be passed as the `contactValue` which will be a String, so we should be OK to remove it. Or